contributing-guides/style-guide.md

```diff
@@ -195,22 +195,22 @@ When Chinese words, Latin words and Arabic numerals are written in the same sent
 
 The following guidelines are applied to Chinese (zh) and traditional Chinese (zh_TW):
 
-1. Place one space before/after English words and numbers.  
```

Comment:
The trailing spaces generate a line break. You can replace then with `<br>`.

Reply:
Wouldn't using numbered list (1,2,3, etc) already generate a line break by default without using a whitespace?

> I checked this page, It seems fine to me.

Reply:
Github line break without trailing spaces. Left side is main branch. Right side is your branch.
![Screenshot from 2022-10-09 18-16-18](https://user-images.githubusercontent.com/46962923/194751260-625a5b57-9faa-4760-bd60-b755bffda610.png)

Reply:
> Github doesn't generate line breaks without trailing spaces. Left side is your branch. Right side is main branch. ![Screenshot from 2022-10-09 18-16-18](https://user-images.githubusercontent.com/46962923/194751260-625a5b57-9faa-4760-bd60-b755bffda610.png)

Oh, Then I will replace it with `<br>` as you mentioned.